x/twap/logic.go

```diff
@@ -95,10 +95,12 @@ func recordWithUpdatedAccumulators(record types.TwapRecord, newTime time.Time) t
 	// thus it is treated as the effective spot price until the new time.
 	// (As there was no change until at or after this time)
 	// TODO: Think about overflow
```

Comment:
@AlpinYukseloglu you're working on this correct? Is there any issue we can reference here?

We should really avoid naked TODOs.

Reply:
I think this is being tracked here: https://github.com/osmosis-labs/osmosis/issues/2412